internal/services/containers/kubernetes_cluster_node_pool_resource.go

```diff
@@ -213,13 +213,12 @@ func resourceKubernetesClusterNodePoolSchema() map[string]*pluginsdk.Schema {
 			ForceNew:     true,
 			RequiredWith: []string{"enable_node_public_ip"},
 		},
-
 		// Node Taints control the behaviour of the Node Pool, as such they should not be computed and
 		// must be specified/reconciled as required
 		"node_taints": {
 			Type:     pluginsdk.TypeList,
 			Optional: true,
-			ForceNew: true,
+			ForceNew: false,
```

Comment:
If an attribute is no longer valid we usually remove it instead of setting it to `false`
```suggestion
```